### PR TITLE
Adds triadic census function to new `triads` module

### PR DESCRIPTION
**Pull request by jfinkels**
Fixes issue #191.

I took one of the implementations and one of the test functions I found in the referenced issue and updated them, mostly for style.

### CODE REVIEW

**Comment by chebee7i** [comment]
Can you also update the release notes?

**Comment by jfinkels** [comment]
Done.

File: networkx/algorithms/triads.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    The triadic census is a count of how many of the 16 possible types of
+    triads are present in a directed graph.
+
+    Parameters
+    ----------
+
+    G : digraph
+       A NetworkX DiGraph
+
+    Returns
```
Comment (chebee7i) [inline networkx/algorithms/triads.py]: Can you remove the newlines before the first item in each section of the docstring? So these should be:

```
Parameters
----------------
G : digraph
```

**Comment by jfinkels** [comment]
Removed initial newlines after subsection headers in documentation strings.

**Pull request merged**